src/inc-lib-init.php

```diff
@@ -1560,8 +1562,13 @@ function lovd_getVariantInfo ($sVariant, $sTranscriptID = '', $bCheckHGVS = fals
             if ($bCheckHGVS) {
                 return false;
             }
-            $aResponse['errors']['EPOSITIONFORMAT'] = 'Too many positions are given for variant type substitution.';
-            return $aResponse;
+            if ($aVariant['earliest_start'] != $aVariant['earliest_end']) {
+                // If the two positions are not the same, the variant is not fixable.
+                $aResponse['errors']['ETOOMANYPOSITIONS'] = 'Too many positions are given for variant type substitution.';
+                return $aResponse;
+            }
+            // If the positions are the same, the variant can safely be interpreted and fixed accordingly.
+            $aResponse['warnings']['WTOOMANYPOSITIONS'] = 'Too many positions are given for variant type substitution.';
```

Comment:
By returning a warning if both positions are the same, and an error if the two positions are different, fixHGVS can easily recognise what she can and cannot do to fix this variant.

Reply:
This entire block (from "Checking all type-specific format requirements" to the last `elseif()`) really needs comments (in the code). I'm getting lost there. There's so many `if()s` on the position fields, that I can't see anymore what kind of variants each block is aiming for. I can deduce it from the error messages, but having examples documented in the code makes it easier to spot the relevant blocks, plus, it becomes easier to spot possible omissions.

Reply:
Good point! Has been done :)